.docker/compose_ci.yaml

```diff
@@ -10,6 +10,7 @@ services:
         - 10023:8080
       environment:
         - JDBC_DATABASE_URL=jdbc:postgresql://db:5432/core?user=bob&password=changeit
+        - ION_CORE_SECRET_KEY=t8abumA3JBJrd7q0LuN3nSzKGBfslOYb
```

Comment:
What is this secret key?

Reply:
The secret key used to sign access_tokens used in import links.

Reply:
Can it be on a public repo?

Reply:
This one is used for testing so, it's fine. The real secret, however, will be set as an environment variable of the GCP VM instances which will be secret.